homeassistant/components/esphome/voice_assistant.py

```diff
@@ -190,6 +248,51 @@ async def run_pipeline(
             tts_audio_output = (
                 "raw" if self.device_info.voice_assistant_version >= 2 else "mp3"
             )
+
+            if self.device_info.voice_assistant_version >= 3:
```

Comment:
Does this mean that all new voice assistants will work with the VAD? Should we make it an option to enable/disable ?

Reply:
Its really hard to know tbh. I did need to set the bits_per_sample to 32 on the echo and proto for it to work. (https://github.com/esphome/esphome/pull/4884)

Perhaps I can set it to v3 only if someone uses a supported mic with 32bit sampling scaled down.

A config option to actually enable it could be good. Would that be on or off by default though I guess? Also until proven otherwise I could enforce it can only be off if you dont use 32bit for the mic.